### ISSUE TITLE
Repository name is misspelled?

### ISSUE THREAD

**Issue by joelanford** [operator-framework/kubernetes-webooks-framework#134]
Add an `h`:

`s/kubernetes-webooks-framework/kubernetes-webhooks-framework/`

Also, should this have a `java-` prefix? This is the first time we've really done something non-Go, so I'm not sure how best to differentiate.

**Comment by csviri** [comment]
omg... fixed the typo.. I have locally still the old name (this was renamed at some point)

> Also, should this have a java- prefix? This is the first time we've really done something non-Go, so I'm not sure how best to differentiate.

agree, that should be somehow differentiated,  I don't like the spring boot starter name either:

https://github.com/operator-framework/operator-framework-spring-boot-starter

any suggestions @metacosm @andreaTP ?

**Comment by csviri** [comment]
maybe rename:
`operator-framework-spring-boot-starter` ->  `java-operator-sdk-spring-boot-starter` ?
`kubernetes-webhooks-framework` -> `java-kubernetes-webhooks-framework` ?

**Comment by csviri** [comment]
cc @shawkins @manusa

**Comment by andreaTP** [comment]
My 2 cents:

> Also, should this have a java- prefix? 

I would consolidate on a short name that is idiomatic for the project, i.e. `josdk` (capitalized or not ...)

`operator-framework-spring-boot-starter` -> `josdk-spring-boot-starter`
`kubernetes-webhooks-framework` -> `josdk-webhooks`

**Comment by csviri** [comment]
> My 2 cents:
> 
> > Also, should this have a java- prefix?
> 
> I would consolidate on a short name that is idiomatic for the project, i.e. `josdk` (capitalized or not ...)
> 
> `operator-framework-spring-boot-starter` -> `josdk-spring-boot-starter` `kubernetes-webhooks-framework` -> `josdk-webhooks`

yeah, this was an alternative also was thinking, I was more hesitant on josdk-webhooks, but tbh probably this would be better.

**Comment by csviri** [comment]
if @metacosm agrees, I'm fine to remove those. (To the ones Andrea suggested)

**Comment by metacosm** [comment]
Agreed on the `josdk`-prefixed names.

**Comment by csviri** [comment]
maybe would be better for webhooks `josdk-webhooks-framework`, webhooks is quite general, I for sure would associate for some callbacks or something for controllers, rather than something else.

**Comment by andreaTP** [comment]
From what I see around, ppl on GH tend to use the shorter possible name for projects.
Usually, keeping the bare name for the "main" project and adding suffixes for the satellites e.g.: `-starter`.

Also, this project is under the organization `operator-framework` and I tend to think that most ppl expect the main projects here to be "framework"s.

That said, no strong opinions here, just defending an opinionated POV.

**Comment by csviri** [comment]
ok convinced :) and  renamed. If we receive some complain we can always re-evaluate.

**Comment by joelanford** [comment]
Thanks! Looks great.
